Magento_Catalog/layout/catalog_product_view.xml

```diff
@@ -0,0 +1,97 @@
+                    </referenceBlock>
+
+                    <block class="Magento\Catalog\Block\Product\View" name="product.column.right.benefits" template="Magento_Catalog::product/view/benefits.phtml" />
+                </referenceContainer>
+            </container>
+
+            <container name="product.tabs" htmlClass="product-view__tabs" htmlTag="div">
+                <referenceBlock name="reviews.tab">
+                    <block class="Magento\Review\Block\Product\View\ListView" name="product.tabs.reviews" template="Magento_Review::product/view/list.phtml"/>
+
+                    <block class="Magento\Theme\Block\Html\Pager" name="product_review_list.toolbar"/>
+                </referenceBlock>
+            </container>
+
+            <container name="product.cms.content" htmlTag="div" htmlClass="additional-content product-view__additional-content">
+                <block class="Magento\Catalog\Block\Product\View" name="product.cms.content.text" template="Magento_Catalog::product/view/content.phtml" />
+            </container>
+        </container>
+
+        <!--Moved blocks-->
+        <move element="product.container" destination="content" before="-" />
+        <move element="product.info.media" destination="product.column.left" />
+
+        <move element="page.main.title" destination="product.column.right.upper" before="-" />
+        <move element="product.info.sku" destination="product.column.right.upper" after="page.main.title" />
+        <move element="product.info.price" destination="product.column.right.upper" after="page.main.title" />
+        <move element="product.info.type" destination="product.column.right.upper" after="product.info.price" />
+        <move element="product.info.review" destination="product.column.right.review" />
+
+        <move element="product.info.options.wrapper" destination="product.column.right.options" />
+        <move element="product.info.main" destination="product.wrapper"/>
+        <move element="product.info" destination="product.info.main" after="product.info.overview" />
+        <move element="product.info.addtocart" destination="product.column.right.extra.actions" />
+        <move element="product.info.social" destination="product.column.right.extra.actions" />
+        <move element="view.addto.wishlist" destination="product.info.social" before="-" />
+        <move element="view.addto.compare" destination="product.info.social" before="product.info.mailto" />
+
+        <move element="product.info.details" destination="product.tabs" />
+        <move element="product.review.form" destination="reviews.tab" after="product_review_list.toolbar" />
+
+        <move element="content.aside" destination="product.container" />
+
+        <move element="product.cms.content" destination="product.container" after="content.aside" />
+
+        <!-- Removed blocks -->
+        <referenceBlock name="product.info.addto" remove="true" />
+    </body>
+</page>
```

Comment:
If is over 120 chars please split to multiple lines

Reply:
splited